doc/source/whatsnew/v0.23.0.txt

```diff
@@ -898,6 +898,48 @@ yourself. To revert to the old setting, you can run this line:
 
   pd.options.display.max_columns = 20
 
+.. _whatsnew_0230.boxplot.sharexy:
+
+Optional sharing of x/y-axis by pandas.DataFrame().groupby().boxplot()
```

Comment:
I updated the docstring and the the part in v0.23.0.txt.

Reply:
Can you move this to 0.23.1.txt now?

Reply:
And I think the example can be trimmed down. I'd rather update the docstring for `DataFrame.plot.box` since that's a longer-term thing. The release note are for people checking changes. They'll see that it now supports `sharey`, and click through if interested.